editoast/editoast_schemas/src/rolling_stock.rs

```diff
@@ -45,7 +48,7 @@ editoast_common::schemas! {
     loading_gauge_type::schemas(),
 }
 
-pub const ROLLING_STOCK_RAILJSON_VERSION: &str = "3.2";
+pub const ROLLING_STOCK_RAILJSON_VERSION: &str = "3.3";
```

Comment:
Why bumbing this value? Towed rolling stocks is another objects.

> [!NOTE]
> Bumping this value requires to create a migration to apply this new version to the RS in DB. It also requires to adapt core.

This property is only here to do a small check when importing / exporting objects. We haven't set up this system everywhere (train schedules). I'm not sure if we should still use it or not.

Reply:
So do I drop that change? Or do you think we should make a different versioning for towed rolling stock?

Reply:
I think we can drop this field for towed RS. And maybe we should discuss keeping it for other objects later?

Reply:
I was also thinking of removing the `railjson_version` field from the towed rolling stock.

Reply:
I'll keep it for consistency.